docs/blocks/creating-dynamic-blocks.md

```diff
@@ -83,16 +83,18 @@ registerBlockType( 'my-plugin/latest-post', {
 ```
 {% end %}
 
-Because it is a dynamic block it also needs a server component. The rendering can be added using the `render_callback` property when using the `register_block_type` function.
+Because it is a dynamic block it also needs a server component. The rendering can be added using the `render_callback` property when using the `register_block_type` function. Please note, that the `render_callback` function should be called by the `init` action.
```

Comment:
- Grammar: The comma after "Please note" can be removed.
- `render_callback` is not the responsibility of the block implementer to be called. Was it intended to be `register_block_type` ?